docs/tutorials/1. Getting started.md

```diff
@@ -135,10 +132,10 @@ console.log(voicedProbabilities);
 
 delete outputPyYin;
 
-// free memory 
-essentia.delete();
-// shutdown essentia algorithm instance
+// call internal essentia::shutdown C++ method
```

Comment:
Add a caution here, that you should only use this method at the end of the lifecycle of using essentia. That means, you shouldn't use it if you want to compute another feature in the same context etc.